src/lib/aggregations/get-avg.spec.ts

```diff
@@ -38,15 +38,15 @@ describe('getAvgAggregation', () => {
         expect(result.aggregations.testAggregation.value).toBeDefined()
     })
 
-    it(`should return an error after passing string field`, async () => {
+    it('should return an error after passing string field', async () => {
         const service = app.get(ElasticsearchService)
 
         await service
             .search(HomeDocument, {
                 size: 0,
                 aggregations: {
                     // eslint-disable-next-line @typescript-eslint/no-explicit-any
-                    testAggregation: getAvgAggregation('address') as any
+                    testAggregation: getAvgAggregation('address' as any)
```

Comment:
we can add `<HomeDocument>(...)` here to avoid `any` casts

Reply:
`address` is not number type, so thats why I'm typing this as any

Reply:
for the purpose of this test only